src/EVM/Transaction.hs

```diff
@@ -47,25 +53,63 @@ data Transaction = Transaction {
     txAccessList :: [AccessListEntry],
     txMaxPriorityFeeGas :: Maybe W256,
     txMaxFeePerGas :: Maybe W256
-} deriving Show
+} deriving (Show, Generic)
+instance JSON.ToJSON Transaction where
```

Comment:
same here re: newline

Reply:
Yeah, thanks, good point! I fixed it in all places now :) Will push and resolve!